core/vm/runtime/env.go

```diff
@@ -17,14 +17,16 @@
 package runtime
 
 import (
+	"github.com/celo-org/celo-blockchain/contracts"
 	"github.com/celo-org/celo-blockchain/core/vm"
+	"github.com/celo-org/celo-blockchain/core/vm/context"
```

Comment:
The `vm` package is still coupled to the contract layer, this is importing the context package which is importing the contracts package.

To break this coupling I would move the `Context` object back into the `vm` package because that is where it lives in geth so moving it out of the `vm` package is a departure from geth.